src/linkgraph/linkgraph_gui.cpp

```diff
@@ -341,7 +341,7 @@ void LinkGraphOverlay::DrawStationDots(const DrawPixelInfo *dpi) const
  * Draw a square symbolizing a producer of cargo.
  * @param x X coordinate of the middle of the vertex.
  * @param y Y coordinate of the middle of the vertex.
- * @param size Y and y extend of the vertex.
+ * @param size Y and y extent of the vertex.
```

Comment:
Good catch, but the unedited part still doesn't make sense. Is it supposed to be this? 😄 

```suggestion
 * @param size X and y extent of the vertex.
```